src/lib/Scenes/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkContextArtist.tsx

```diff
@@ -10,7 +10,13 @@ export const ArtworkContextArtistFragmentContainer = createFragmentContainer<{
 }>(
   props => {
     return (
-      <Join separator={<Spacer my={2} />}>
+      <Join
```

Comment:
Next I'll pick up the ticket about adding the rest of the contexts, and this logic can be moved into a central location.